index.html

```diff
@@ -3915,6 +4098,119 @@ <h2>
             more information.
           </p>
         </section>
+        <section>
+          <h2>
+            <dfn data-lt="requestBillingAddress(addressParts)">
+              requestBillingAddress()
+            </dfn> method
+          </h2>
+          <aside class="note">
+            <p>
+              The {{PaymentMethodChangeEvent/requestBillingAddress()}} method
+              allows the developer to incrementally request additional pieces of
+              the billing address that were not initially requested in
+              {{ PaymentOptions.requestBillingAddressParts }}.
```

Comment:
Just noting that the usage of {{X.Y}} in other parts of the spec is wrong... I need to fix those ☺️ The {{X.Y}} means "access **static** method/attribute Y on X" - while {{X / Y}} means "access the **instance** method/attribute Y of X"

```suggestion
              {{ PaymentOptions/requestBillingAddressParts }}.
```